pages/user/[username].tsx

```diff
@@ -8,25 +8,37 @@ import { ContributorsProfileType } from "components/molecules/ContributorHoverCa
 import HubPageLayout from "layouts/hub-page";
 import HubLayout from "layouts/hub";
 import ProfileLayout from "layouts/profile";
+import { useFetchUser } from "lib/hooks/useFetchUser";
 
 const Contributor = () => {
   const router = useRouter();
   const { username } = router.query;
   const contributorLogin = username as string;
 
-  const { data: contributor, isLoading: contributorLoading, isError } = useSingleContributor(contributorLogin);
+  const {
+    data: contributor,
+    isLoading: contributorLoading,
+    isError: contributorError
+  } = useSingleContributor(contributorLogin);
+  const { data: user, isLoading: userLoading, isError: userError } = useFetchUser(contributorLogin);
 
+  const isLoading = contributorLoading || userLoading;
```

Comment:
Just use `isLoading` from line 23. We don't want to block the UI on the contributor call to finish.

```suggestion
  const isLoading = contributorLoading || userLoading;
```